<launch>
  <!--  
    Right now, the android app is hard coded to look for /cmd_vel as the input
    topic.
   -->
  <!-- relay in topic_tools package does not work with multimaster.. -->
  <node pkg="rocon_python_utils" name="cmd_vel_relay" type="topic_relay" args="/teleop/cmd_vel mobile_base/commands/velocity geometry_msgs/Twist">
  </node>

</launch>
